nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/document/AddEntryToMultiValuedPropertyTest.java

```diff
@@ -18,15 +18,18 @@
  */
 package org.nuxeo.ecm.automation.core.operations.document;
 
+import static java.lang.Thread.sleep;
```

Comment:
Please don't `import static` `Thread.sleep`, it's unexpected as we never do it anywhere else.